File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/DismissButton.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class DismissButton{

	@SerializedName("buttonRenderer")
	private ButtonRenderer buttonRenderer;

	public void setButtonRenderer(ButtonRenderer buttonRenderer){
		this.buttonRenderer = buttonRenderer;
	}

	public ButtonRenderer getButtonRenderer(){
		return buttonRenderer;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/DismissButton.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class DismissButton : Parcelable, Serializable {
    @SerializedName("buttonRenderer")
    var buttonRenderer: ButtonRenderer? = null
    override fun toString(): String {
        return "DismissButton{" +
                "buttonRenderer = '" + buttonRenderer + '\'' +
                "}"
    }
}